website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Provisioning Configuration"
+linkTitle: "Provisioning"
+weight: 10
+---
+
+The [Provisioner CRD]({{< ref "../reference/provisioner-crd.md" >}}) provides two sections for constraining nodes. 

```

Comment:
There's actually three (taints). I recommend avoiding framing things "n-ways" to do something unless there's something fundamental about why n will only ever be n. This is the type of detail that tends to not survive the test of time.

Reply:
only "spec.requirements" and "spec.provider" change the calls to EC2 right?